addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4557,6 +4559,8 @@ export class OdooEditor extends EventTarget {
             this._applyCommand("insert", text);
         } else if (odooEditorHtml) {
             const fragment = parseHTML(odooEditorHtml);
+            const editedLinks = fragment.querySelectorAll('.oe_edited_link');
```

Comment:
I was going to say you could also include `this.options.renderingClasses` in there because we don't want to copy these either for sure. But actually I realized that `oe_edited_link` is included in it. So instead of removing `oe_edited_link` specifically, you should remove all rendering classes from that array.